documentation/src/test/java/org/hibernate/search/documentation/search/query/QueryDslIT.java

```diff
@@ -306,6 +307,31 @@ public void explain_elasticsearch() {
 		} );
 	}
 
+	@Test
+	public void tookAndTimedOut_elasticsearch() {
```

Comment:
See, you even put the example in the integration test about the query DSL (not the projection DSL) :)